lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_mercator_coordinate_system.py

```diff
@@ -0,0 +1,60 @@
+# (C) British Crown Copyright 2016, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Test function :func:`iris.fileformats._pyke_rules.compiled_krb.\
+fc_rules_cf_fc.build_mercator_coordinate_system`.
+
+"""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# import iris tests first so that some things can be initialised before
+# importing anything else
+import iris.tests as tests
+
+import numpy as np
+
+import iris
+from iris.cube import Cube
+from iris.coord_systems import Mercator
+from iris.fileformats._pyke_rules.compiled_krb.fc_rules_cf_fc import \
+    build_mercator_coordinate_system
+from iris.tests import mock
+
+
+class TestBuildMercatorCoordinateSystem(tests.IrisTest):
+    def test_valid(self):
+        cf_grid_var = mock.Mock(
```

Comment:
This test (good though it is) does not exercise `inverse_flattening`. It might be worth adding a near-identical test where your mock `cf_grid_var` includes a `semi_major_axis` and an `inverse_flattening`.